### PR TITLE
Updated clusterMatch docs

### PR DESCRIPTION
**Pull request by hzxuzhonghu**

### CODE REVIEW

File: networking/v1alpha3/envoy_filter.proto [hunk 1]
```diff
@@ -517,12 +517,14 @@ message EnvoyFilter {
...
     // The service port for which this cluster was generated.  If
     // omitted, applies to clusters for any port.
+    // **Note:** for inbound cluster, it is the service target port.
     uint32 port_number = 1;
 
     // The fully qualified service name for this cluster. If omitted,
     // applies to clusters for any service. For services defined
     // through service entries, the service name is same as the hosts
     // defined in the service entry.
+    // **Note:** for inbound cluster, this is ignorred.
```
Comment (hzxuzhonghu) [inline networking/v1alpha3/envoy_filter.proto:527]: After inbound cluster name has been changed, this is meaningless for inbound.

Comment (ramaraochavali) [reply networking/v1alpha3/envoy_filter.proto:527]: nit: /s/ignorred/ignored

Comment (hzxuzhonghu) [reply networking/v1alpha3/envoy_filter.proto]: done

**Comment by hzxuzhonghu** [comment]
@ramaraochavali Can you approve? Since you touch the implementation recently.

**Review by ramaraochavali — APPROVED** [review]

**Comment by hzxuzhonghu** [comment]
@nrjpoddar can you approve

**Review by howardjohn — APPROVED** [review]

**Review by nrjpoddar — APPROVED** [review]

**Pull request merged**